os/board/rtl8721csm/src/component/os/tizenrt/rtk_netmgr.c

```diff
@@ -473,6 +477,15 @@ trwifi_result_e wifi_netmgr_utils_deinit(struct netdev *dev)
 	if (ret == RTK_STATUS_SUCCESS) {
 		g_mode = RTK_WIFI_NONE;
 		wuret = TRWIFI_SUCCESS;
+		if (saved_scan_list) {
+			rtw_mfree((unsigned char *)saved_scan_list, sizeof(ap_scan_list_s) * scan_number);
+			saved_scan_list = NULL;
+		}
+		scan_number = 0;
+		if (scan_timer.timer_hdl != NULL) {

```

Comment:
I think its better to stop and delete timer before freeing saved_scan_list..

Reply:
Done as suggested.